styles/whisper.json

```diff
@@ -432,11 +432,6 @@
 				}
 			}
 		},
-		"border": {
-			"color": "var(--wp--preset--color--tertiary)",
-			"style": "solid",
-			"width": "1vh 1vw"
-		},
```

Comment:
I believe this is part of the design, as mentioned [here](https://github.com/WordPress/twentytwentythree/issues/147#issuecomment-1239673282).

Reply:
Thanks for pointing. Missed that.
I would rather revert it to a fixed width of 0.5rem instead of vh and vw, because they give uneven left-right and top-bottom spacing. Let me know your thoughts.

Reply:
This makes sense to me 👍 Or perhaps we could make this fluid, using something like `max(1vw, 0.5rem)`?

@colorful-tones @beafialho, tagging you for your thoughts here too.

Reply:
In order to include this in Beta 1, I've gone ahead and used `max(1vw, 0.5rem)`. Please feel free to open another PR to make additional changes!